### PR TITLE
New TC for In case HMI sends GetURLs and no apps registered SDL must …

### PR DESCRIPTION
**Pull request by istoimenova**
To run the script you need to replace
user_modules/shared_testcases/testCasesForPolicyTable.lua
user_modules/shared_testcases/testCasesForPolicyTableSnapshot.lua
user_modules/shared_testcases/testCasesForBuildingSDLPolicyFlag.lua
from #32. Common functions are under review.

@maria-mih please review the script according to template.

### CODE REVIEW

File: test_scripts/Polices/Policy_Table_Update/011_ATF_P_TC_HMI_sends_GetURLs_no_app_registered.lua [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+config.defaultProtocolVersion = 2
+
+--[[ General Settings for configuration ]]
+Test = require('connecttest')
+require('cardinalities')
+require('user_modules/AppTypes')
+local mobile_session = require('mobile_session')
+
+--[[ Preconditions ]]
+testCasesForPolicyTable:flow_PTU_SUCCEESS_EXTERNAL_PROPRIETARY()
```
Comment (istoimenova) [inline test_scripts/Polices/Policy_Table_Update/011_ATF_P_TC_HMI_sends_GetURLs_no_app_registered.lua]: @istoimenova Call of function should be updated to be included in Test.

File: test_scripts/Polices/Policy_Table_Update/011_ATF_P_TC_HMI_sends_GetURLs_no_app_registered.lua [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+function Test:Precondition_UnregisterApp()
+  local CorIdURAI = self.mobileSession:SendRPC("UnregisterAppInterface", {})
+  EXPECT_HMINOTIFICATION("BasicCommunication.OnAppUnregistered",
+    {appID = self.applications[config.application1.registerAppInterfaceParams.appName], unexpectedDisconnect = false})
+  EXPECT_RESPONSE("UnregisterAppInterface", {success = true , resultCode = "SUCCESS"})
+end
+
+-- Request PTU
+testCasesForPolicyTable:trigger_PTU_user_request_update_from_HMI()
```
Comment (istoimenova) [inline test_scripts/Polices/Policy_Table_Update/011_ATF_P_TC_HMI_sends_GetURLs_no_app_registered.lua]: @istoimenova Call of function should be updated to be included in Test.

File: test_scripts/Polices/Policy_Table_Update/011_ATF_P_TC_HMI_sends_GetURLs_no_app_registered.lua [hunk 3]
```diff
@@ -0,0 +1,85 @@
+---------------------------------------------------------------------------------------------
+-- Requirements summary:
+-- In case HMI sends GetURLs and no apps registered SDL must return only default url
+-- [HMI API] GetURLs request/response
+--
+-- Description:
+-- SDL should request PTU in case user requests PTU
```
Comment (maria-mih) [inline test_scripts/Polices/Policy_Table_Update/011_ATF_P_TC_HMI_sends_GetURLs_no_app_registered.lua]: Please check description

File: test_scripts/Polices/Policy_Table_Update/011_ATF_P_TC_HMI_sends_GetURLs_no_app_registered.lua [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+
+  EXPECT_HMIRESPONSE(RequestId,{result = {code = 0, method = "SDL.GetURLS", urls = endpoints} } )
+  :Do(function(_,data) 
+    if(is_app_esxist == false) then
+      self:FailTestCase("endpoints for application doesn't exist!")
+    end
+  end)
+end
+
+return Test
```
Comment (maria-mih) [inline test_scripts/Polices/Policy_Table_Update/011_ATF_P_TC_HMI_sends_GetURLs_no_app_registered.lua]: Maybe add Postconditions?

**Comment by istoimenova** [comment]
@LuxoftAKutsan please merge.

**Pull request merged**